build.gradle

```diff
@@ -263,13 +287,13 @@ jacocoTestReport {
     getAdditionalSourceDirs().from(sourceSets.main.allJava.srcDirs)
 
     reports {
-        xml.enabled = true // coveralls plugin depends on xml format report
-        html.enabled = true
+        xml.required = true // coveralls plugin depends on xml format report
+        html.required = true
     }
 }
 
 wrapper {
-    gradleVersion = '7.5.1'
+    gradleVersion = '8.1.1'
```

Comment:
There seems to be a mismatch here, this says 8.1.1 but the wrapper properties says 8.2.1.  I think this should be 8.2.1.